docs/en/pact-reference.md

```diff
@@ -599,6 +612,218 @@ Examples of valid keyset JSON productions:
 
 ```
 
+### Namespaces {#namespaces}
```

Comment:
Why is this here and not above (or the above below down here?) Cf. guards and capabilities which are one section each/in one place.